tools/syz-repro/repro.go

```diff
@@ -20,24 +20,25 @@ import (
 )
 
 var (
-	flagConfig = flag.String("config", "", "configuration file")
+	flagConfig = flag.String("config", "", "manager configuration file (manager.cfg)")
 	flagCount  = flag.Int("count", 0, "number of VMs to use (overrides config count param)")
 	flagDebug  = flag.Bool("debug", false, "print debug output")
 )
 
 func main() {
 	os.Args = append(append([]string{}, os.Args[0], "-v=10"), os.Args[1:]...)
 	flag.Parse()
+	if len(flag.Args()) != 1 || flagConfig == nil {
```

Comment:
`*flagConfig == ""`

Reply:
Indeed, I confused string with nil slices that have interesting behaviors.